src/id/doujindesu/src/eu/kanade/tachiyomi/extension/id/doujindesu/DoujinDesu.kt

```diff
@@ -252,17 +252,17 @@ class DoujinDesu : ParsedHttpSource(), ConfigurableSource {
         Genre("Yuri"),
     )
 
-    private class CategoryNames(categories: Array<Category>) :
-        Filter.Select<Category>("Category", categories, 0)
-
-    private class OrderBy(orders: Array<Order>) : Filter.Select<Order>("Order", orders, 0)
+    private class AuthorFilter : Filter.Text("Author")
+    private class CharacterFilter : Filter.Text("Karakter")
+    private class CategoryNames(categories: Array<Category>) : Filter.Select<Category>("Kategori", categories, 0)
+    private class OrderBy(orders: Array<Order>) : Filter.Select<Order>("Urutkan", orders, 0)
     private class GenreList(genres: List<Genre>) : Filter.Group<Genre>("Genre", genres)
     private class StatusList(statuses: Array<Status>) : Filter.Select<Status>("Status", statuses, 0)
 
     private fun basicInformationFromElement(element: Element): SManga {
         val manga = SManga.create()
         element.select("a").let {
-            manga.title = it.attr("title")
+            manga.title = element.select("h3.title").text()
```

Comment:
`.selectFirst()`

Reply:
manga.title = element.select("h3.title").selectFirst()
Correct?

Reply:
Nope, `element.selectFirst("h3.title")!!.text()`

Not sure if you need `!!`, it is in any case safe to use here since title has to be set.

Reply:
Ahh got it, sorry i almost forget everything because i didn't touch coding for 2 years after my Laptop broken